### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by Bachibouzouk**
## About
Improve the documentation as I deploy WAM for the first time.

### CODE REVIEW

File: doc/getting_started.rst [hunk 1]
```diff
@@ -27,8 +27,8 @@ Setup conda environment with required packages via:
 
   conda env create -f environment.yml
 
-Afterwards, applications can be "plugged-in" by simply cloning application into wam folder
```
Comment (Bachibouzouk) [inline doc/getting_started.rst:30]: This was confusing as there is a `wam` folder inside the `WAM` repository

File: doc/getting_started.rst [hunk 2]
```diff
@@ -64,7 +64,7 @@ Environment Variables
 WAM-Server needs at least the following environment variables:
 
 - CONFIG_PATH: Path to configuration file (mainly includes database configurations, see :ref:`configuration`)
-- DJANGO_SECRET_KEY: Set a secret key for *settings.py* (from *settings.py*: SECURITY WARNING: keep the secret key used in production secret!)
```
Comment (Bachibouzouk) [inline doc/getting_started.rst:66]: It seemed to me that this was redundant :)

File: doc/getting_started.rst [hunk 3]
```diff
@@ -64,7 +64,7 @@ Environment Variables
...
+- DJANGO_SECRET_KEY: Set a secret key for *settings.py* (SECURITY WARNING: keep the secret key used in production secret!)
 - WAM_APPS: Apps which shall be loaded within *INSTALLED_APPS*. Additionally, individual app settings are loaded (see :ref:`app_settings`).
```
Comment (Bachibouzouk) [inline doc/getting_started.rst:80]: I think theses variables could be more descriptive in terms of the formatting of their possible values : 
as I understood it, `WAM_APPS` should contain the name of the folder in which each app is located separated by commas?

For `DJANGO_SECRET_KEY` : is the description "Set a secret key for *settings.py*" suppose to explain the purpose of this variable or it ask the user to set a secret key (could it be anything then?)

**Review by Bachibouzouk — COMMENTED** [review]
@henhuy : I left a few comments/questions along my changes

File: doc/getting_started.rst (hunk 2 above)
Comment (henhuy) [reply doc/getting_started.rst:66]: you are right!
I changed code to use only env vars `CONFIG_PATH` and `WAM_APPS`.
My plan is to move `WAM_APPS` also into config file and only set env `CONFIG_PATH`

File: doc/getting_started.rst (hunk 3 above)
Comment (henhuy) [reply doc/getting_started.rst:80]: Double yes!
- WAM_APPS is list of additional apps (foldernames)
- `DJANGO_SECRET_KEY` can be any str, but should be a strong password - there is also a [django secret key generator](https://www.miniwebtool.com/django-secret-key-generator/)

File: install_requirements.py [hunk 4]
```diff
@@ -2,10 +2,11 @@
...
 base_dir = os.path.abspath(os.path.curdir)
+pip_path = subprocess.check_output(['which', 'pip']).decode().strip('\n')
 apps = os.environ['WAM_APPS'].split(',')
 for app in apps:
     req_file = os.path.join(base_dir, app, 'requirements.txt')
     if os.path.isfile(req_file):
         subprocess.call(
-            ['/opt/conda/envs/django/bin/pip', 'install', '-r', req_file]
+            [pip_path, 'install', '-r', req_file]
         )
```
Comment (Bachibouzouk) [inline install_requirements.py:12]: my pip wasn't under that path (I have Ubuntu 18.04)

**Pull request merged**